Yoast/javascript
# Move Russian stemming data to morphology data file

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Moves the Russian stemming data to a newly created Russian morphology data file.

## Relevant technical choices:
 * I also enabled support in `getLanguagesWithWordFormSupport` because that's needed to test whether the morphology data file gets properly picked up in the content analysis app.
* Enabling morphology support meant that the behavior of some unit tests that previously assumed no morphology support for Russian changed. Those were adapted to reflect the new situation. For the changes in the Russian full text specs, I had a look together with Hanna to check that we don't get any unexpected results. Where there were any changes, we recognized more word forms than previously (as expected) but still not all forms actually present in the text (also not wholly unexpected for the first iteration of the stemmer).

## Test instructions

This PR can be tested by following these steps:

* Check the unit tests
  * Run `yarn test` and make sure that all tests still pass.
  * Make sure that the coverage is still the same as before the change.
* Test in the content analysis app
  * Start the content analysis app by going to `javascript/apps/content-analysis` and run `yarn start`.
  * In the content analysis app, make sure that the locale is set to the right language and that the toggle for morphology support is switched on.
  * Add a text of at least 150 words.
  * Set a keyphrase and use various (supported) forms of that keyphrase in the text.
  * Make sure that all forms get recognized in the keyphrase density assessment (click on the eye marker next to the result and confirm that you can see all forms marked below).

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-394
